### PR TITLE
implement basic info on user edit page

### PR DESCRIPTION
**Pull request by samanthaming**

### CODE REVIEW

**Comment by samweatherly** [comment]
:+1:

File: app/views/users/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<h1>Basic Information (edit)</h1>
```
Comment (zibs) [inline app/views/users/edit.html.erb:1]: If this is the same form as the users/new.html.erb, maybe we should considering make this file and the new file into a _form.html.erb partial?

**Comment by zibs** [comment]
LGTM +1 :dancer:

File: app/views/users/edit.html.erb (hunk 1 above)
Comment (samanthaming) [inline app/views/users/edit.html.erb:1]: create account is different from create profile....so NO

Comment (zibs) [inline app/views/users/edit.html.erb:1]: :8ball:

**Pull request merged**